mithril-common/src/entities/signed_entity_type.rs

```diff
@@ -139,6 +149,10 @@ impl SignedEntityType {
                 hasher.update(&epoch.to_be_bytes());
                 hasher.update(&block_number.to_be_bytes())
             }
+            SignedEntityType::CardanoDatabase(db_beacon) => {
```

Comment:
This case could be handled jointly with `SignedEntityType::CardanoImmutableFilesFull`

Reply:
Done ✅